safe/impact_function/impact_function.py

```diff
@@ -1766,6 +1765,7 @@ def post_process(self, layer):
         # Set the layer title
         layer_title(layer)
 
+        from safe.definitions.post_processors import post_processors
```

Comment:
So we can't import post_processors at the beginning of file as usual ? If yes, you should put a comment why you want to import not at the beginning.  Same comment where you import not at the beginning.